tests/expressions/tests.py

```diff
@@ -777,26 +777,26 @@ def test_case_in_filter_if_boolean_output_field(self):
                 output_field=BooleanField(),
             ),
         )
-        self.assertSequenceEqual(qs, [self.example_inc.ceo, self.foobar_ltd.ceo, self.max])
+        self.assertCountEqual(qs, [self.example_inc.ceo, self.foobar_ltd.ceo, self.max])
 
     def test_boolean_expression_combined(self):
         is_ceo = Company.objects.filter(ceo=OuterRef('pk'))
         is_poc = Company.objects.filter(point_of_contact=OuterRef('pk'))
         self.gmbh.point_of_contact = self.max
         self.gmbh.save()
-        self.assertSequenceEqual(
+        self.assertCountEqual(
             Employee.objects.filter(Exists(is_ceo) | Exists(is_poc)),
             [self.example_inc.ceo, self.foobar_ltd.ceo, self.max],
         )
-        self.assertSequenceEqual(
+        self.assertCountEqual(
             Employee.objects.filter(Exists(is_ceo) & Exists(is_poc)),
             [self.max],
         )
-        self.assertSequenceEqual(
+        self.assertCountEqual(
```

Comment:
This looks unnecessary, because we compare with a one-item list :thinking: (the same in L791).

Reply:
Correct, but stylistically I think it would be odd to switch assertion based on number of items.